mobile/.gitignore

```
@@ -0,0 +1,17 @@
+
+
+#amplify
+amplify/\#current-cloud-backend
+amplify/.config/local-*
+amplify/mock-data
+amplify/backend/amplify-meta.json
+amplify/backend/awscloudformation
```

if nothing from the `amplify/` folder is being uploaded (I didn't see anything from it), then maybe it's simpler to just ignore the whole folder.
```suggestion
amplify/
```